deeplay/components/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .mlp import MultiLayerPerceptron
 from .cnn import *
+from .rnn import *
```

Comment:
I try to have `import *` only from folders, and exact imports from files. The alternative is to define __all__ = [...] in the file. Otherwise any imports inside that module becomes exported by that module (for example deeplay.rnn.Optional)